prExplorer/API/appsettings.Development.json

```diff
@@ -7,5 +7,6 @@
   },
   "ConnectionStrings": {
     "DefaultConnection": "Data Source=prExplorer.db"
-  }
+  },
+  "TokenKey":"0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Remember, especially if your code would be open-source, to not expose any keys publicly